### PR TITLE
38 frontend login

### PR DESCRIPTION
**Pull request by allthesignals**
Addresses #38 by adding login functionality.

Includes commits from @bfreeds branch.

### CODE REVIEW

File: client/app/controllers/projects.js [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
-            //     return value.packages[0].statuscode === "Active"
-            // }
-        })
+        return this.model.filter((project) => {
+            console.log('project being filtered', project)
+            // if (project.primaryApplicant === 'Brandyn Friedly') { return true}
+
+            return project.packages.some((projectpackage) => {
+                if (projectpackage.statuscode === 'Saved' || projectpackage.statuscode === 'Package Preparation') {
+                    console.log("TRUEEE", project)
```
Comment (godfreyyeung) [inline client/app/controllers/projects.js]: should nix these, haha

Comment (allthesignals) [reply client/app/controllers/projects.js]: :)

File: client/mirage/scenarios/default.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      primaryApplicant: "Brandyn Friedly",
+      packages: [
+        {
+          type: 'pas',
+          statuscode: 'Saved'
+        }
+      ]
+    },
+    {
+      name: "Bagel Factory",
```
Comment (godfreyyeung) [inline client/mirage/scenarios/default.js]: Omg I loooovee this mirage data you made up. So fitting for all of us 😂

File: client/app/controllers/projects.js [hunk 3]
```diff
@@ -1,33 +1,28 @@
...
-                    } else { return false }
-            })
-
-        })
-    }
-
-    get upcomingProjects () {
-        // TODO: write filter for projects with no pas package objects
-    }
+  get upcomingProjects () {
```
[the remark this answers is not in the document]
Comment (allthesignals) [reply client/app/controllers/projects.js:24]: @godfreyyeung this is a Q for @bfreeds

**Pull request closed without merging**